target/linux/ramips/image/mt76x8.mk

```diff
@@ -809,10 +809,16 @@ TARGET_DEVICES += tplink_re220-v2
 
 define Device/tplink_re305-v1
   $(Device/tplink-safeloader)
-  IMAGE_SIZE := 6016k
+  IMAGE_SIZE := 7680k
+  KERNEL_SIZE := 6016k
   DEVICE_MODEL := RE305
   DEVICE_VARIANT := v1
   DEVICE_PACKAGES := kmod-mt76x2
+  DEVICE_COMPAT_VERSION := 1.1
```

Comment:
This will only force the user to use `sysupgrade -n`, which allows to install (oversized) sysupgrade images to old partition layout. Change it to 2.0 to tell the user to perform a new install.